django/forms/fields.py

```diff
@@ -1250,6 +1250,8 @@ def to_python(self, value):
     def bound_data(self, data, initial):
         if self.disabled:
             return initial
+        if data is None:
+            return InvalidJSONInput()
```

Comment:
I'm not sure if it's a correct solution, before db19619545dd99a1d2502c72974d79eca33acff7 we returned `json.dumps(None)` i.e. `'null'`. IMO it's expected.
```suggestion
        if data is None:
            return data
```